### DISCUSSION TITLE
Table spanning multiple pages and with merged cells

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#2286] [Q&A] [answered]
Attached is a table with merged cells that spans multiple pages, which I did using LaTeX package `tabularray`.

[tables_merged_cells.pdf](https://github.com/typst/typst/files/12781997/tables_merged_cells.pdf)

I would like to know whether `typst`, at its current development stage, can do tables like the one shown in the attached PDF file.

Thanks!

**Comment by Dherse** [comment, marked as answer]
There is a package in the registry called "tablex" made by @PgBiel, it should be exactly what you're looking for.

**Reply by PgBiel** [reply]
([tablex*](https://github.com/PgBiel/typst-tablex))

**Reply by Dherse** [reply]
God dang autocorrect 😠

**Reply by psads-git** [reply]
Thanks, @Dherse and @PgBiel, for your help!

**Reply by Dherse** [reply]
You're welcome, hope you have fun with typst 🎉
